packages/babel-plugin/src/css-prop/__tests__/behaviour.test.ts

```diff
@@ -182,15 +182,15 @@ describe('css prop behaviour', () => {
       import '@compiled/react';
 
       const base = { color: 'black' };
-      const top = \` color: red; \`;
+      const top = \` marginTop: 10px; \`;
```

Comment:
Why was this changed?

Reply:
because I created another test case to [test deduping atomic group](https://github.com/atlassian-labs/compiled/pull/1354/files#diff-69558aafb510bb07e99ed2dc72efeb29d5ff2370b0a3910c2a996c71f4081fa9R737). So I updated this one to not have duplicated `color`.